### ISSUE TITLE
conv2 is wrong?

### ISSUE THREAD

**Issue by AnthonyF333** [polarisZhao/PFLD-pytorch#30]
class PFLDInference(nn.Module):
    def __init__(self):
        super(PFLDInference, self).__init__()

        self.conv1 = nn.Conv2d(
            3, 64, kernel_size=3, stride=2, padding=1, bias=False)
        self.bn1 = nn.BatchNorm2d(64)
        self.relu = nn.ReLU(inplace=True)

        self.conv2 = nn.Conv2d(
            64, 64, kernel_size=3, stride=1, padding=1, bias=False)
        self.bn2 = nn.BatchNorm2d(64)
        self.relu = nn.ReLU(inplace=True)

@polarisZhao I think conv2 is a depth-wise convolution instead of a normal convolution, right?

**Comment by polarisZhao** [comment]
yes, you're right. Thanks a lot. I will fix it.
